src/implementations/wrap/wrap.ts

```diff
@@ -202,10 +213,15 @@ export const wrap = async ({
   const hashMap = merkleHashmap(individualDocumentHashes);
```

Comment:
Actually this is one of the most expensive operation. We can skip by this entirely. 

I'm thinking if we should just have another function to handle non-batched documents, its quite straightforward and doesnt need to copy to intermediate directory.... Can straightaway copy individual files over from input dir to output dir, running the batch function individually.

Reply:
In fact after phase 1, phase 2 and 3 is not needed anymore...

Reply:
changed =)